staging/src/k8s.io/apiserver/pkg/storage/interfaces.go

```diff
@@ -243,3 +240,22 @@ type Interface interface {
 	// Count returns number of different entries under the key (generally being path prefix).
 	Count(key string) (int64, error)
 }
+
+// GetOptions provides the options that may be provided for storage get operations.
+type GetOptions struct {
+	// IgnoreNotFound determines what is returned if the requested object is not found. If true, a zero object is
```

Comment:
nit: can you please shorten the lines?

Reply:
Sure, I'll keep them under 100char/line which seems to be the norm for this file.